hil/cli.py

```diff
@@ -411,7 +411,7 @@ def node_register(node, subtype, *args):
         "ipmi", <hostname>, <ipmi-username>, <ipmi-password>
     """
     obm_api = "http://schema.massopencloud.org/haas/v0/obm/"
-    obm_types = ["ipmi", "mock"]
+    obm_types = ["ipmi", "mock", "netns"]
```

Comment:
why is this here?

Reply:
working on multiple branches. will remove it in next commit. done.